src/redux/comments/comments.reducer.js

```diff
@@ -0,0 +1,42 @@
+import {
+	SET_COMMENTS,
+	SET_RATE,
+	SET_COMMENTS_LOADING,
+	SET_UPDATING_COMMENT
+} from './comments.types';
+
+export const initialState = {
+	commentsLoading: false,
+	updatingComment: null,
+	comments: [],
+	rate: 0
```

Comment:
Why do we need this rate if I have seen the same one in the local state of the component?

Reply:
This rate is for summary product rate value from the backend(all votes), and rate in the local state is for input, when user is leaving the comment